sdk/storage/Azure.Storage.Blobs/api/Azure.Storage.Blobs.netstandard2.0.cs

```diff
@@ -257,6 +257,8 @@ public partial class BlobContainerItem
     public partial class BlobContainerProperties
     {
         internal BlobContainerProperties() { }
+        public string DefaultEncryptionScope { [System.Runtime.CompilerServices.CompilerGeneratedAttribute] get { throw null; } }
+        public bool? DenyEncryptionScopeOverride { [System.Runtime.CompilerServices.CompilerGeneratedAttribute] get { throw null; } }
```

Comment:
The naming for this property is a bit odd. According to the guidelines, we should try to use affirmative phrasing (https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-type-members)
✓ DO name Boolean properties with an affirmative phrase (CanSeek instead of CantSeek). Optionally, you can also prefix Boolean properties with "Is," "Can," or "Has," but only where it adds value.

What about CanOverrideEncryptionScope?

Reply:
The name of the property in the service is DenyEncryptionScopeOverride.  I think it would be confusing for the customer if we name it something else in BlobContainerProperties.

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/9192

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/9192